app/imports/components/ActionItems/ActionItemsBoard/ActionItems.jsx

```diff
@@ -0,0 +1,240 @@
+                            open={open}
+                            assignee={assignee}
+                            closeMessage={closeMessage}
+                            toggleActionItem={() => this.showToggleActionItemModal(_id)}
+                            isModerator={isModerator}
+                            idToRemove={idToRemove}
+                            sprintId={sprintId}
+                            onData={onData}
+                            handlers={handlers}
+                            wrappedData={wrappedData}
+                        />,
+                )}
+
+                <AddActionItem
+                    open={showAddActionItemModal}
+                    onSubmit={this.addActionItem}
+                    error={errorAdd}
+                    onClose={this.hideAddActionItemModal}
+                />
+
+                <CloseOrReopenActionItem
+                    open={showToggleActionItemModal}
+                    onSubmit={this.toggleModalActionItem}
+                    error={errorToggle}
+                    onClose={this.hideToggleActionItemModal}
+                    projectId={projectId}
+                />
+
+                <Snackbar
+                    open={openSnackbar}
+                    message="New action item has been added!"
+                    autoHideDuration={4000}
+                    onRequestClose={() => closeSnackBar(sprintId, onData, handlers, wrappedData)}
+                />
+
+                <Snackbar
+                    open={openToggleSnackbar}
+                    message="Changes saved!"
+                    autoHideDuration={4000}
+                    onRequestClose={() => closeSnackBar(sprintId, onData, handlers, wrappedData)}
+                />
+            </div>
+        );
+    }
+}
+
+ActionItems.defaultProps = {
+    errorToggle: null,
+    errorAdd: null,
+    idToRemove: '',
+    openSnackbar: false,
+    openToggleSnackbar: false,
+};
+
+ActionItems.propTypes = {
+    sprintId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    addActionItem: PropTypes.func.isRequired,
+    closeSnackBar: PropTypes.func.isRequired,
+    wrappedData: PropTypes.func.isRequired,
+    onData: PropTypes.func.isRequired,
+    toggleActionItem: PropTypes.func.isRequired,
+    isMember: PropTypes.bool.isRequired,
+    isModerator: PropTypes.bool.isRequired,
+    isClosed: PropTypes.bool.isRequired,
+    actionItems: PropTypes.arrayOf(
+        PropTypes.shape({
+            _id: PropTypes.string.isRequired,
+            text: PropTypes.string.isRequired,
+            startDate: PropTypes.instanceOf(Date).isRequired,
+            endDate: PropTypes.instanceOf(Date).isRequired,
+            assigneeId: PropTypes.string.isRequired,
+            assignee: PropTypes.shape({
+                name: PropTypes.string,
+                avatar: PropTypes.string,
+            }).isRequired,
+        }),
+    ).isRequired,
+    handlers: PropTypes.arrayOf(
```

Comment:
Meteor and composer are necessary to properly working functions in index.js with onData wrapper. Cant figure better solution